modules/apps/configuration-admin/configuration-admin-web/src/main/resources/META-INF/resources/edit_configuration.jsp

```diff
@@ -230,27 +230,27 @@ renderResponse.setTitle(categoryDisplayName);
 					<liferay-util:dynamic-include key='<%= "com.liferay.configuration.admin.web#/edit_configuration.jsp#" + configurationModel.getFactoryPid() + "#post" %>' />
 
 					<c:if test="<%= !configurationModel.isReadOnly() %>">
-						<aui:button-row>
-							<c:choose>
-								<c:when test="<%= configurationModel.hasScopeConfiguration(configurationScopeDisplayContext.getScope()) %>">
-									<aui:button name="update" type="submit" value="update" />
-								</c:when>
-								<c:otherwise>
-									<aui:button name="save" type="submit" value="save" />
-								</c:otherwise>
-							</c:choose>
-
-							<aui:button cssClass="ml-3" href="<%= redirect %>" name="cancel" type="cancel" />
-						</aui:button-row>
-
-						<c:if test="<%= Validator.isNotNull(configurationModel.getLiferayLearnMessageKey()) && Validator.isNotNull(configurationModel.getLiferayLearnMessageResource()) %>">
-							<div class="float-right">
+						<div class="align-items-center d-flex justify-content-between">
+							<aui:button-row>
```

Comment:
I'm not sure if this is using `btn-group` but maybe we could add it later to have smoother spacing between the buttons. For example https://clayui.com/docs/components/button-group.html, you can edit the example by adding `spaced` in `ClayButton.Group`.

Reply:
Yep, this isn't using btn-group... at some point it may make sense to add a Clay Button Group taglib though, to replace all these usages.